try:
    a = 5 / 0
except:
    raise Exception("Have error while deleting")
print("!")
